programs/stake_api/src/stake_instruction.rs

```diff
@@ -123,6 +139,24 @@ pub fn process_instruction(
                 &syscall::rewards::from_keyed_account(&rest[0])?,
             )
         }
+        StakeInstruction::Deactivate => {
+            if rest.len() != 0 {
+                Err(InstructionError::InvalidInstructionData)?;
+            }
+            me.deactivate_stake(&syscall::current::from_keyed_account(&me)?)
```

Comment:
rest.len() should be exactly 1 and current comes from &rest[0]

Reply:
👍